RELEASE/scripts/autoscend/quests/level_13.ash

```diff
@@ -585,7 +585,12 @@ boolean L13_towerNSContests()
 				if(crowd3Insufficient()) auto_beachCombHead("spooky");
 				if(crowd3Insufficient()) buffMaintain($effect[Spooky Hands]);
 				if(crowd3Insufficient()) buffMaintain($effect[Spooky Weapon]);
-				if(crowd3Insufficient()) buffMaintain($effect[Dirge of Dreadfulness (Remastered)]);
+				// at this point, an example list of songs is phat loot / polka / celerity / madrigal
+				if(crowd3Insufficient()) {
+					shrugAT($effect[Dirge of Dreadfulness (Remastered)]);
```

Comment:
`shrugAT` has a check for if we have the skill. Even with the velour vaqueros equipped, `has_skill` for dirge remastered returns false. So I don't think this will work as is. It won't shrug as `shrugAT` will return early since it thinks we don't have the skill

Suggestion is to use no param for remastered, aka `shrugAT()`, and then call `shrugAT($effect[Dirge of Dreadfulness])` for the normal version of it. That way something will get shrugged, if needed, for the remastered version. There is risk that remastered will get shrugged for the normal version. I think that is a low risk, but could move the remastered version to the end of `ATSongList()` to make it so that is the last song to get checked. Meaning the last song to get shrugged

Reply:
moving it to the end sounds good -- under normal circumstances it'll never get obtained in the first place, so it's reasonable to say we never want to lose it.

Reply:
I think the no param version won't work either -- `have_skill(to_skill($effect[none]))` is going to be false, so it'll drop out at the same point.